IdentityCore/src/oauth2/aad_v2/MSIDAADV2TokenResponse.m

```diff
@@ -28,9 +28,16 @@
 
 @implementation MSIDAADV2TokenResponse
```

Comment:
I think we also added B2C token. Can you double check if it needs same changes?

Reply:
B2C implementation is currently empty so I believe we are good there.

Reply:
Added B2C implementation but not sure if we need to override the account type. In the current code, it will be set to "MSIDAccountTypeMSSTS".

Reply:
It should still be MSSTS